systemtest/src/test/java/io/strimzi/systemtest/KafkaClusterTest.java

```diff
@@ -363,6 +373,50 @@ public void testForUpdateValuesInConfigMap() {
         }
     }
 
+    @Test
+    @Resources(value = "../examples/templates/cluster-controller", asAdmin = true)
+    @OpenShiftOnly
+    public void testSendMessages() {
+        String topicName = "test-topic";
+        Oc oc = (Oc) this.kubeClient;
+        int messagesCount = 5;
+        oc.namespace();
+        String clusterName = "openshift-my-cluster";
+        oc.newApp("strimzi-ephemeral", map("CLUSTER_NAME", clusterName));
+        oc.waitForStatefulSet(zookeeperStatefulSetName(clusterName), 1);
+        oc.waitForStatefulSet(kafkaStatefulSetName(clusterName), 3);
+        String bootstrapServers = "";
+        for (int i = 0; i < 3; i++) {
+            String kafkaPodJson = oc.getResourceAsJson("pod", kafkaPodName(clusterName, i));
+            bootstrapServers = bootstrapServers + JsonPath.parse(kafkaPodJson).read("$.status.podIP").toString() + ":9092,";
+        }
+        bootstrapServers = bootstrapServers.substring(0, bootstrapServers.length() - 1);
+
+        String yaml = getContent(new File("../examples/configmaps/topic-controller/kafka-topic-configmap.yaml"), node -> {
+            JsonNode metadata = node.get("metadata");
+            ((ObjectNode) metadata).put("name", topicName);
+            JsonNode labels = metadata.get("labels");
+            ((ObjectNode) labels).put("strimzi.io/cluster", clusterName);
+            JsonNode data = node.get("data");
+            ((ObjectNode) data).put("name", topicName);
+        });
+        oc.createContent(yaml);
+        oc.waitForDeployment(topicControllerName(clusterName));
```

Comment:
We could replace this with a `@Topic` annotation, consumed by the `StrimziRunner` which would create this CM for you. Doing it that way would allow you to very easily reuse for other tests which needed a topic.